tool_collections/fgbio/.shed.yml

```
@@ -0,0 +1,15 @@
+owner: iuc
+remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fgbio
+homepage_url: https://github.com/fulcrumgenomics/fgbio
+categories:
+  - SAM
+suite:
+  name: suite_fgbio
+  description: "Tools for high-throughput sequece data analysis from fulcrum genomics."
+  remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fgbio
+  homepage_url: https://github.com/fulcrumgenomics/fgbio
+  long_description: |
+    A set of tools to analyze genomic data with a focus on high-throughput sequenceing.
+  include_repositories:
+    - name: fgbio_findswitchbackreads
+      owner: iuc
```

`planemo` is correct about the deployment failure. Can you open a PR containing something like:

```suggestion
owner: iuc
remote_repository_url: https://github.com/galaxyproject/tools-iuc/tree/main/tool_collections/fgbio
homepage_url: https://github.com/fulcrumgenomics/fgbio
description: fgbio
long_description: |
    A set of tools to analyze genomic data with a focus on high-throughput sequenceing
categories:
  - SAM
type: unrestricted
auto_tool_repositories:
  name_template: "{{ tool_id }}"
  description_template: "Wrapper for fgbio function: {{ tool_name }}."
suite:
  name: "suite_fgbio"
  description: "Tools for high-throughput sequence data analysis from fulcrum genomics."
  long_description: |
      A set of tools to analyze genomic data with a focus on high-throughput sequenceing
```